src/components/Tabs/_tabs.scss

```diff
@@ -0,0 +1,53 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-tabs-component {
+  @include text($body2);
+
+  width: 100%;
+
+  &__tabs-list {
+    display: flex;
+    gap: 1rem;
+    overflow: hidden; /* Prevent overflow */
+    padding: 0 $padding-md;
+    position: relative;
+  }
+
+  &__tabs-trigger {
+    background: none;
+    border: none;
+    color: $button-hover;
+    cursor: pointer;
+    padding: 0 0 $padding-sm;
+    transition: background 0.2s ease;
+
+    &:hover {
+      color: $pure-black;
+    }
+  }
+
+  &__tabs-trigger[data-state='active'] {
+    border-bottom: 2px solid $pure-black;
+    color: $pure-black;
+    font-variation-settings: 'wght' 600;
+
+    &:hover {
```

Comment:
can we get a better focus state for keyboard navigation of these Tabs?  Currently it's the default blue outline which looks pretty bad.  I believe  something like this from breadcrumb 
```  &:focus-visible {
        border-radius: 5px;
        outline: 1px solid $medium-gray;
      }
```